Automattic/wp-calypso
# State: Move Jetpack connection selectors to global selectors directory

This PR migrates the `jetpack/connection` selectors to the global selectors directory, which is the recommended place to add new selectors. See #5954 for more information, reasoning and rationale.

To test:
* Checkout this branch
* Verify all state tests pass: `npm run test-client client/state/`
